run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-cd `dirname $0`
+cd "$(dirname "$0")"
```

Comment:
Nonetheless, as written this is broken. It specifies a posix shbang but uses bashisms.